h/models/organization.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+
+
+import sqlalchemy as sa
+import slugify
+
+from h.db import Base
+from h.db import mixins
+
+
+ORGANIZATION_NAME_MIN_CHARS = 1
+ORGANIZATION_NAME_MAX_CHARS = 25
+ORGANIZATION_LOGO_MAX_CHARS = 10000
+
+
+class Organization(Base, mixins.Timestamps):
+    __tablename__ = 'organization'
+
+    id = sa.Column(sa.Integer, autoincrement=True, primary_key=True)
+
+    name = sa.Column(sa.UnicodeText(), nullable=False, index=True)
+
+    logo = sa.Column(sa.UnicodeText())
+
+    def __init__(self, **kwargs):
+        super(Organization, self).__init__(**kwargs)
```

Comment:
I see that this `__init__()` method is copied from `Group`, but I have no idea why `Group` has an `__init__()` method like this! This doesn't do anything (Python will behave exactly as this `__init__()` method says by default, if the class has no `__init__()`.) Our other existing model classes besides `Group` don't have this either. Can we remove this method?

Reply:
oh yes you are right! originally i had it taking an id as an arg for the factory but since i got rid of the factory I don't need this here at all anymore...